### ISSUE TITLE
[chat] fund historical price data

### ISSUE THREAD

**Issue by andrey-jef** [haxj/haxj.github.io#1]
Hi haxj,

I've just passed by your post related to vn fund. I'm interested in your EOD price data. If it's possible, could you please share the source where did get the historical data of these funds, like VESAF and DCDS? Or how did you scrape them? Thanks and keep publishing.

https://haxj.github.io/posts/vietnam-funds/

**Comment by haxj** [comment]
Hi andrey, the historical data are published on the website of the fund management companies. You can find them in the excel files or in the fund's performance chart.

**Comment by andrey-jef** [comment]
I agree that we can read the historical price data on performance chart or the daily nav change excel file. I'm curious how did you collect these data points? by manually downloading the excel file then reorganizing them into a csv file to be visualized in your chart? Or did u run a selenium script? I also want to collect these historical data to run my own analysis.

I also found a paid 3rd party data provider ([wifeed](https://www.wigroup.vn/wifeed)), but just not worth yet for my pet project at the moment.
